File: JSBASIC/123_fibonacci.js
// 123	123_fibonacci.js	Generate and print the Fibonacci series up to n terms.

let n = 10;
let a = 0, b = 1;
console.log("Fibonacci series up to", n, "terms:");
for (let i = 1; i <= n; i++) {
    process.stdout.write(a + " ");
    let next = a + b;
    a = b;
    b = next;
}
console.log();

File: JavaBasic/123_fibonacci.java
// 123	123_fibonacci.java	Generate and print the Fibonacci series up to n terms.
public class _123_fibonacci {
    public static void main(String[] args) {
        int n = 10, a = 0, b = 1;
        System.out.print("Fibonacci (" + n + " terms): ");
        for (int i = 1; i <= n; i++) { System.out.print(a + " "); int next = a+b; a=b; b=next; }
        System.out.println();
    }
}
